assisted-events-scrape/utils/counters.py

```diff
@@ -0,0 +1,39 @@
+import threading
+from datetime import datetime
+from datetime import timedelta
+
+
+class ErrorCounter:
+    def __init__(self):
+        self.lock = threading.Lock()
+        self.value = 0
+
+    def inc(self) -> None:
+        with self.lock:
+            self.value += 1
+
+    def get_errors(self) -> None:
```

Comment:
Type doesn't match - Expected type 'None', got 'int' instead 

```suggestion
    def get_errors(self) -> union[int, None]:
```